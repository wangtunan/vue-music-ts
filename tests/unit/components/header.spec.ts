import { shallowMount, Wrapper } from '@vue/test-utils'
import MHeader from '@/components/header/index.vue'

describe('header.vue', () => {
  let wrapper: Wrapper<Vue>
  beforeEach(() => {
    wrapper = shallowMount(MHeader, {
      stubs: ['router-link']
    })
  })
  it('match shapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
  it('mounted success', () => {
    expect(wrapper.exists()).toBe(true)
  })
})
