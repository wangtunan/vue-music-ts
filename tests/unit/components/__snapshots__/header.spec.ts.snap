// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header.vue match shapshot 1`] = `
<div class="m-header"><img src="@/assets/images/logo.png" alt="" class="header-logo">
  <h1 class="header-title">Chicken Music</h1>
  <router-link-stub to="/user" tag="div" class="header-mine"><i class="icon-mine"></i></router-link-stub>
</div>
`;
